README.md

```diff
@@ -32,3 +53,35 @@ var v3 = new Vector(3, 4)
 console.log(v3.getLength());
 // => 5
 ```
+```
+class Vector{
+  constructor(x,y){
+  this.x = x;
+  this.y = y;
+}
+
+plus(vec){
+  let sum1 = this.x + vec.x;
+  let sum2 = this.y + vec.y;
+  return `x:${sum1}, y:${sum2}`
+}
+
+minus(vec){
+  let sum1 = this.x - vec.x;
+  let sum2 = this.y - this.y;
+  return `x:${sum1}, y:${sum2}`
+}
+
+getLength(){
+  return Math.hypot(this.x, this.y)
+}
+}
+
+
+let v1 = new Vector(1,2)
+let v2 = new Vector(2,3)
+let v3 = new Vector(3,4)
+console.log(v1.plus(v2));
+console.log(v1.minus(v2));
+console.log(v3.getLength());
+```
```

Comment:
Really great job! 👍 
Small note: variables that are classes should begin with an uppercase.